CorsixTH/Lua/dialogs/resizables/file_browsers/load_game.lua

```diff
@@ -27,7 +27,7 @@ local UILoadGame = _G["UILoadGame"]
 function UILoadGame:UILoadGame(ui, mode)
   local treenode = FilteredFileTreeNode(ui.app.savegame_dir, ".sav")
   treenode.label = "Saves"
-  self:UIFileBrowser(ui, mode, _S.load_game_window.caption:format(".sav"), 295, treenode, true)
+  self:UIFileBrowser(ui, mode, _S.load_game_window.caption:format(".sav"), 295, treenode, true, _S.load_game_window.load_button)
```

Comment:
```suggestion
  self:UIFileBrowser(ui, mode, _S.load_game_window.caption:format(".sav"), 295,
      treenode, true, _S.load_game_window.load_button)
```
Split the line like below?